scripts/deploy-docs

```diff
@@ -66,4 +66,5 @@ git config user.name "${GITHUB_USER_NAME}"
 git remote set-url origin https://$GITHUB_USER_NAME:$GITHUB_TOKEN@github.com/solarwinds/nova-docs.git
 git add .
 git commit -m "add docs" --allow-empty
+git pull -X theirs
```

Comment:
what was this for exactly?

Reply:
This will pull any changes and if there are any merge conflicts uses theirs.  It will fix some issues with that error we see.